README.md

```diff
@@ -1,46 +1,29 @@
-## 18F Guides Template
+## Joining 18F
 
-This is a skeleton repo containing the
-[CFPB/DOCter](https://github.com/CFPB/DOCter)-based
-[Jekyll](http://jekyllrb.com/) template for
-[18F Guides](http://18f.github.io/guides/).
+This is the repository for the page about [Joining 18F](https://pages.18f.gov/joining-18f/). This is an [18F Guide](https://github.com/18F/guides), built on the [18F Guides Template](https://github.com/18F/guides-template).
 
-### Generating the site/hosting locally
+The application form itself lives in a separate repository: https://github.com/18F/joining-18f-app.
 
-You will need [Ruby](https://www.ruby-lang.org) ( > version 2.1.5 ). You may
-consider using a Ruby version manager such as
-[rbenv](https://github.com/sstephenson/rbenv) or [rvm](https://rvm.io/) to
-help ensure that Ruby version upgrades don't mean all your
-[gems](https://rubygems.org/) will need to be rebuilt.
+### Send feedback
 
-On OS X, you can use [Homebrew](http://brew.sh/) to install Ruby in
-`/usr/local/bin`, which may require you to update your `$PATH` environment
-variable:
+If you find something confusing or spot an error, please let us know by [filing an issue](https://github.com/18F/joining-18f/issues).
 
-```shell
-$ brew update
-$ brew install ruby
-```
+### Install and edit this guide on your computer
+
+If you want to run your own copy of this guide, you'll need [Ruby](https://www.ruby-lang.org) ( > version 2.1.5 ). For complete instructions, see the README for the [18F Guides Template](https://github.com/18F/guides-template/blob/18f-pages/README.md).
 
-To create a new guide and serve it locally, where `MY-NEW-GUIDE` is the name
-of your new repository:
+#### Cloning and serving the guide locally
+
+To serve this guide locally:
 
 ```shell
-$ git clone git@github.com:18F/guides-template.git MY-NEW-GUIDE
-$ cd MY-NEW-GUIDE
-$ ./go init
```

Comment:
I got an error when I tried to use ./go init, so I removed it. Also, I think some other, more recent READMEs omitted it, so it seemed safe.